spec/features/users/03_user_signs_out_spec.rb

```diff
@@ -1,23 +1,22 @@
 require 'rails_helper'
+require 'support/user_helper'
```

Comment:
I feel it would be better to require this `user_helper` in your `rails_helper` file, since you'll probably be using the `user_sign_in` method in almost all your tests! I'll come over and show you a code example of how we have that set up in our internal apps, it's a pretty small change.